docs/openapi-backend.yaml

```diff
@@ -0,0 +1,48 @@
+openapi: 3.1.0
+info:
+  title: io-backend FIMS REST API
+  version: 0.1.0
+paths:
```

Comment:
can you add a short description for each spec?

Reply:
we will expand these specs (that are roughly draft, right now) as the project will be developed